solidity/scripts/request-relay-entry-with-callback.js

```diff
@@ -1,10 +1,9 @@
-const crypto = require("crypto")
 const KeepRandomBeaconServiceImplV1 = artifacts.require("KeepRandomBeaconServiceImplV1.sol");
 const KeepRandomBeaconService = artifacts.require('KeepRandomBeaconService.sol');
 
 // Example usage:
-// truffle exec ./scripts/request-relay-entry-with-callback.js yourContractAddress "callbackMethodName" callbackGas
-// truffle exec ./scripts/request-relay-entry-with-callback.js 0x9F57C01059057d821c6b4B04A4598322661C934F "callback(uint256)" 20000
+// truffle exec ./scripts/request-relay-entry-with-callback.js yourContractAddress callbackGas
+// truffle exec ./scripts/request-relay-entry-with-callback.js 0x9F57C01059057d821c6b4B04A4598322661C934F 20000
```

Comment:
I think this won't work, the number of params has changed. I've changed the script to this:
```
let entryFeeEstimate = await contractInstance.entryFeeEstimate(process.argv[5]);
    let tx = await contractInstance.methods['requestRelayEntry(address,uint256)'](
      process.argv[4],
      process.argv[5],
      {value: entryFeeEstimate}
    )
```
and then the request went trough

Reply:
Good catch! Fixed in https://github.com/keep-network/keep-core/pull/1532/commits/767add64f9002e5e736ed13a9f5d2e957e908646

Reply:
we also need to change:
`await contractInstance.entryFeeEstimate(process.argv[6])` to
`await contractInstance.entryFeeEstimate(process.argv[5])`

Reply:
🤦‍♂ Done.